arches/app/templates/javascript.htm

```diff
@@ -959,6 +959,7 @@
             "label": "{{ concept_collection.label | escapejs }}",
             "id": "{{ concept_collection.id }}"
         },{% endfor %}]'
+        cookie-prefix="{{ app_settings.SESSION_COOKIE_NAME }}"
```

Comment:
Could you remove this line since we're no longer using cookies for this interaction?

Reply:
@chrabyrd - good catch. I've removed the offending line.